legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/api/GitLabProjectConfigurationApi.java

```diff
@@ -401,6 +407,49 @@ public List<MetamodelDependency> getMetamodelDependencies()
         return ProjectStructure.getProjectStructure(projectConfiguration).getAvailableGenerationConfigurations();
     }
 
+    @Override
+    public ProjectConfigurationStatusReport getProjectConfigurationStatus(String projectId)
+    {
+        return new ProjectConfigurationStatusReport()
+        {
+            @Override
+            public boolean isProjectConfigured()
+            {
+                return getProjectConfiguration(projectId) != null;
+            }
+
+            @Override
+            public List<String> getReviewIds()
+            {
+                if (!isProjectConfigured())
+                {
+                    try
+                    {
+                        GitLabProjectId gitLabProjectId = parseProjectId(projectId);
+                        MergeRequestFilter mergeRequestFilter =  new MergeRequestFilter();
+                        mergeRequestFilter.setProjectId(gitLabProjectId.getGitLabId());
+                        mergeRequestFilter.setTargetBranch(getDefaultBranch(gitLabProjectId));
+                        Stream<MergeRequest> mergeRequests = PagerTools.stream(withRetries(() -> getGitLabApi().getMergeRequestApi().getMergeRequests(mergeRequestFilter, ITEMS_PER_PAGE)));
+                        List<MergeRequest> projectConfigurationMergeRequests = mergeRequests.filter(mergeRequest -> mergeRequest.getSourceProjectId().equals(gitLabProjectId.getGitLabId()) && mergeRequest.getSourceBranch().contains(GitLabProjectApi.PROJECT_CONFIGURATION_WORKSPACE_ID_PREFIX)).collect(Collectors.toList());
+                        if (projectConfigurationMergeRequests.size() > 0)
+                        {
+                            return projectConfigurationMergeRequests.stream().map(mergeRequest -> mergeRequest.getIid().toString()).collect(Collectors.toList());
```

Comment:
You need to make sure that mergeRequest.getIid() is not null before calling toString. There's an method inherited from BaseGitLabApi just for this purpose: toStringIfNotNull.